templates/taxi/manufacturer_delete.html

```diff
@@ -0,0 +1,11 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h1>Delete manufacturer {{ manufacturer.name }}</h1>
+  <br>
+  <h5>Are you sure you want to delete {{ manufacturer.name }}?</h5>
+  <form action="" method="post">
+    {% csrf_token %}
+    <input type="submit" value="Yes" class="btn btn-danger">
+  </form>
```

Comment:
and here add a button `No` or `Cancel`, please. Let's make your code more aesthetic 🐱